UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/FileShareService.cs

```diff
@@ -355,6 +357,9 @@ private async Task<bool> ProcessBatchFile(IEnumerable<string> uri, string downlo
                 }
                 HttpResponseMessage httpResponse = await fileShareServiceClient.CallFileShareServiceApi(HttpMethod.Get, payloadJson, accessToken, item, CancellationToken.None, queueMessage.CorrelationId);
 
+                var requestUri = new Uri(httpResponse.RequestMessage.RequestUri.ToString()).GetLeftPart(UriPartial.Path);
+                var serverValue = httpResponse.Headers.Server.ToString().Split('/').First();
```

Comment:
Kindly move this line 361 inside if condition like below
if (httpResponse.IsSuccessStatusCode)
            {
                var serverValue = httpResponse.Headers.Server.ToString().Split('/').First();

Reply:
I did changes accordingly.